treeherder/seta/settings.py

```diff
@@ -1,3 +1,9 @@
+# repos that SETA supports
+SETA_PROJECTS = [
+    'mozilla-inbound',
+    'autoland'
+]
```

Comment:
Another way of doing this would be to add them as properties to the repositories table, similar to what Perfherder did:
https://github.com/mozilla/treeherder/blob/c5e53a3a90bcd99a7c719bfb0cccd1dc615b551f/treeherder/model/models.py#L91